src/mon/MonMap.cc

```diff
@@ -775,6 +775,28 @@ seastar::future<> MonMap::build_initial(const crimson::common::ConfigProxy& conf
 
 #else  // WITH_SEASTAR
 
+int MonMap::init_with_monaddrs(const std::list<entity_addrvec_t> addrs,
+                               bool for_mkfs,
+                               std::ostream& errout)
+{
+  int i = 0;
+  for (auto const& addr : addrs) {
+    char n[2];
+    n[0] = 'a' + i;
+    n[1] = 0;
+    string name;
+    name = "noname-";
```

Comment:
Should we generate names, or extract it from MonMap and store it as well for use here?